### ISSUE TITLE
crash: trace_span_tag: Assertion `span' failed.

### ISSUE THREAD

**Issue by grubles** [ElementsProject/lightning#7843]
CLN at 1b2d5acf1638558cce8994f45ace363b04e4d113
Fedora 40 x86_64

```
$ ./builds/lightning/lightningd/lightningd --signet
lightningd: common/trace.c:267: trace_span_tag: Assertion `span' failed.
lightningd: FATAL SIGNAL 6 (version v24.08-386-g1b2d5ac-modded)
0x484842 send_backtrace
        common/daemon.c:33
0x4848c1 crashdump
        common/daemon.c:75
0x78a8e9656cff ???
        ???:0
0x78a8e96af664 ???
        ???:0
0x78a8e9656c4d ???
        ???:0
0x78a8e963e901 ???
        ???:0
0x78a8e963e81d ???
        ???:0
0x78a8e964ed86 ???
        ???:0
0x496234 trace_span_tag
        common/trace.c:267
0x4d1a97 db_query_prepared_canfail
        db/utils.c:145
0x49a1da db_data_version_get
        db/exec.c:48
0x4663fe db_setup
        wallet/db.c:1117
0x46c423 wallet_new
        wallet/wallet.c:100
0x430692 main
        lightningd/lightningd.c:1296
0x78a8e9640087 ???
        ???:0
0x78a8e964014a ???
        ???:0
0x405fe4 ???
        ???:0
0xffffffffffffffff ???
        ???:0
lightningd: Died with signal 6
```

**Comment by grubles** [comment]
Ah looks like there is a PR to patch this up already: https://github.com/ElementsProject/lightning/pull/7840

**Comment by grubles** [comment]
Hrm nope still hit the same crash with that PR.

**Comment by rustyrussell** [comment]
Yes, @cdecker missed this one.  Same fix as #7840 though.  Clearly we need to add USDT to one of the CI instances?
